sklearn/utils/tests/test_optimize.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+from sklearn.utils.optimize import newton_cg
+from scipy.optimize import fmin_ncg
+
+from sklearn.utils.testing import assert_array_almost_equal
+
+
+def test_newton_cg():
+    # Test that newton_cg gives same result as scipy's fmin_ncg
+
+    A = np.random.normal(size=(10, 10))
```

Comment:
Can you please use a local random number generator with a specific seed:
<pre>
rng = np.random.RandomState(0)
A = rng.normal(size=(10, 10))
</pre>